server/handlers/handlers.go

```diff
@@ -43,9 +45,11 @@ type PluginIface interface {
 }
 
 type ActivityHandler struct {
-	plugin PluginIface
-	queue  chan msteams.Activity
-	quit   chan bool
+	plugin               PluginIface
+	queue                chan msteams.Activity
+	quit                 chan bool
+	IgnorePluginHooksMap sync.Map
+	Mutex                sync.Mutex
```

Comment:
Do we need to make the casing consistent?

Reply:
discussed